docs/helm/endpoints_api.md

```diff
@@ -13,6 +13,7 @@
 *  **URL Params**
 
    `ns=[string]` - Namespace
+   `isTopology=[boolean]` - isTopology
```

Comment:
can we have something like "limitInfo" or something instead of "isTopology" ?

Reply:
Is the api used in any other place part from topology ?

Reply:
Updated with `limitInfo`